scripts/prepublish

```diff
@@ -7,6 +7,7 @@ set -eo pipefail
 # Build contracts
 yarn clean
 yarn build
+npx hardhat export-abi
```

Comment:
Take a look at this comment https://github.com/graphprotocol/contracts/issues/643#issuecomment-1194022527

Reply:
That is a lot cleaner, i like it!